indico/modules/events/registration/blueprint.py

```diff
@@ -78,8 +81,14 @@
 _bp.add_url_rule('/manage/registration/<int:reg_form_id>'
                  '/registrations/<int:registration_id>/file/<int:field_data_id>-<filename>', 'registration_file',
                  RHRegistrationDownloadAttachment)
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/create', 'create_registration',
+                 RHRegistrationCreate, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/<int:registration_id>/edit', 'edit_registration',
                  RHRegistrationEdit)
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/email', 'email_registrants',
+                 RHRegistrationEmailRegistrants, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/delete', 'delete_registrations',
+                 RHRegistrationDelete, methods=('DELETE',))
```

Comment:
I'd use POST here. DELETE is more appropriate on RESTful URLs (e.g. `DELETE .../registrants/<id>` but of course that doesn't work for multiple items)